indico/htdocs/sass/partials/_tables.scss

```diff
@@ -55,6 +59,10 @@ td.i-table {
     & ~ td.i-table {
         border-left: none !important;
     }
+
+    &.checkbox-column {
+        width: 2.5em;
```

Comment:
If you just want minimal width here you can simply use the `small-column` class instead of adding a new one